qa/rpc-tests/feature_zip221.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+
+from test_framework.flyclient import (ZcashMMRNode, append, delete, make_root_commitment)
+from test_framework.mininode import (HEARTWOOD_BRANCH_ID, CBlockHeader)
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+    bytes_to_hex_str,
+    hex_str_to_bytes,
+    initialize_chain_clean,
+    start_nodes,
+)
+
+from io import BytesIO
+
+NULL_FIELD = "0000000000000000000000000000000000000000000000000000000000000000"
+CHAIN_HISTORY_ROOT_VERSION = 2010200
```

Comment:
... and change this to:
```suggestion
CHAIN_HISTORY_ROOT_VERSION = 2010100
```

Reply:
I think this is no longer relevant.